ldotel/go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/launchdarkly/go-server-sdk/ldotel
 
-go 1.20
+go 1.23.0
```

Comment:
Make sure you add a `BEGIN_COMMIT_OVERRIDE` block and note the fix: Bump go minimum to 1.23.